Source/WebCore/css/CSSProperties.json

```diff
@@ -4310,7 +4310,7 @@
             }
         },
         "dynamic-range-limit": {
-            "animation-type": "see prose",
+            "animation-type": "not animatable",
```

Comment:
Can you add an "animation-type-comment" explaining why it's currently not animatable?

Reply:
The comment should include that the spec defines it as animatable too